### PR TITLE
feat/WPMBD-202

### PR DESCRIPTION
**Pull request by kevdevbaires**
- Added class for image block
- Added return function on image and paragraph
- Added data fields on image block
- Added validation for different types of blocks and data

### CODE REVIEW

File: application/blocks/heading-block.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+			$this->blocks[$key] = [
+				'blockName' => 'parsed',
+				'block_html' => $this->get_attributes($block)
+			];
+		}
+
+		return $this->blocks;
+	}
+
+	private function get_attributes($block) {
```
Comment (ocorralesRV) [inline application/blocks/heading-block.php:37]: shouldn't we try and move this to the parent class? We've used it in 2 blocks so far

File: application/blocks/heading-block.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		if (empty($heading[0])){
+			return false;
+		}else {
+			$heading = $heading[0];
+		}
+
+		return "<{$heading_tag}>{$heading->textContent}</{$heading_tag}>";
+
+	}
+}
```
Comment (ocorralesRV) [inline application/blocks/heading-block.php:59]: We should add an extra line at the end of the file always.

File: application/blocks/list-block.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace CNET\Bridge\Blocks;
+
+class List_Block extends Block_Parser implements Block_Interface {
+
+	const BLOCK_TYPE = 'core/list';
```
Comment (ocorralesRV) [inline application/blocks/list-block.php]: should we have a constant class that contains all of them? Or do you think this is good enough?

File: application/blocks/heading-block.php (hunk 1 above)
Comment (kevdevbaires) [reply application/blocks/heading-block.php]: @ocorralesRV added function to interface

File: application/blocks/heading-block.php (hunk 2 above)
Comment (kevdevbaires) [reply application/blocks/heading-block.php]: @ocorralesRV added missing line

**Pull request merged**